client/src/components/ImageCarousel/ImageCarousel.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+import Carousel from 'react-material-ui-carousel';
+
+const ImageCarousel = () => {
+  const [images, setImages] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      const data = await fetch('./heroSectionImages.json').then((res) => res.json());
+      setImages(data);
+    })();
+  }, []);
+
+  return (
```

Comment:
The Carousel is not in the center of the page if I change the page scale. Please, see the screenshoot below.
But still, If I visit this page via the usual 100% screen scale everything seems to be OK. 
I`m not sure that this is a bug, but please double-check from your device as sometimes, my Chrome dev tools show me unexpected things.
![image](https://user-images.githubusercontent.com/112622259/198870144-403fb06a-4c88-426f-9dab-a27fca2bdcb1.png)

![image](https://user-images.githubusercontent.com/112622259/198870164-2255c8f8-7b9f-4717-871e-7a75a59b8bff.png)

Reply:
![image](https://user-images.githubusercontent.com/107735322/198877158-f7e10491-b4b1-4cb4-a324-7e0c0b864b65.png)


please check the different breakpoints view using chrome devtools responsive button. as changing the zoom scale is not an accurate way for this )